modules/getting-started/con-techdocs-config-cicd.adoc

```diff
@@ -0,0 +1,27 @@
+[id="con-techdocs-config-cicd_{context}"]
+
+= Configuring CI/CD to generate and publish TecDocs sites
+
+TechDocs reads the static generated documentation files from a cloud storage bucket, such as OpenShift Data Foundation. The documentation site is generated on the CI/CD workflow associated with the repository containing the documentation files. You can generate docs on CI and publish to a cloud storage using the `techdocs-cli` tool.
+
+Supported CI providers include GitHub Actions, CircleCI, or Jenkins. You can use the following example script to create a script for TechDocs publication:
```

Comment:
Where is this coming from? We don't provide support for any of the CI providers.
Almost any  CI system can be used to generate techdocs.

Reply:
This content came from [this link](documented in https://backstage.io/docs/features/techdocs/configuring-ci-cd) that you shared. If it is not accurate for RHDH. Do we need to remove or change anything beyond _Supported CI providers include GitHub Actions, CircleCI, or Jenkins_?

Reply:
the origin document makes it clear that the listed CI is not the only CI that you can use. Maybe I'm translating it wrong, but to me, it looks like the 3 listed CI providers are the only one that you can use. 

We should also be careful about using term "supported" as it might look like this is something that Red Hat provides support for.

Reply:
I now understand that it was probably also the word "including" that was causing the issue. I removed it. Rather than supported, we can say "available" or "compatible". The other option is to remove them altogether.